### DISCUSSION TITLE
What is the reason for a white screen when zed for windows opens?

### DISCUSSION THREAD

**Discussion by zerx-lab** [zed-industries/zed#12071] [Feature Requests]
@JunkuiZhang
![image](https://github.com/zed-industries/zed/assets/161401688/58d10d42-661a-4560-8c8e-f4a2e9854fa3)
 Could you please tell me if this problem can be optimized?

**Comment by JunkuiZhang** [comment]
This is strange, it never hanppens on my end. Have you tried `cargo run --release`?

**Reply by zerx-lab** [reply]
When I open the build --release package, a 500ms white screen flashes. Are you on win11 or win10? I am currently running win10 22H2 system.

**Reply by JunkuiZhang** [reply]
Im on Win11

**Reply by zerx-lab** [reply]
Can you give me a compiled exe file? I will test it on win10. I just compiled it with win11 sdk and it still shows a white screen.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
